xtrabackup/filesystem_utils.py

```diff
@@ -93,3 +93,8 @@ def split_path(path):
 def get_prefixed_file_in_dir(directory, prefix):
     files = glob(''.join([directory, '/', prefix, '*']))
     return files[0]
+
+def get_prefixed_files_in_dir(directory, prefix):
```

Comment:
Looks like duplicated code to me. Just define `get_prefixed_files_in_dir` and remove the first one. Then update the function call in https://github.com/deviantony/xtrabackup-scripts/blob/5eaeec1466d994ea103c24765699321970bfc1b3/xtrabackup/restoration_tools.py#L84 with `backup_archive = filesystem_utils.get_prefixed_file_in_dir(
                archive_repository, prefix)[0]`